server/src/__tests__/constants.js

```diff
@@ -1,5 +1,14 @@
-import constants from '../constants';
+import { distantFuture, nowInUTC } from '../constants';
 
 it('exists', () => {
-  expect(constants).toBeDefined();
+  expect(distantFuture).toBeDefined();
+  expect(nowInUTC).toBeDefined();
 });
+
+it('nowInUTC returns a number', () =>
+  expect(nowInUTC()).toBeGreaterThan(0),
+);
+
+it('distantFuture returns a number', () =>
+  expect(distantFuture).toEqual(2147483647),
```

Comment:
better:
```
expect(distantFuture).toBeGreaterThan(new Date().convertToUnixTime()),
```

Reply:
(make sure it's converted into seconds not milliseconds otherwise the test won't be valid)
maybe add 100 years to the timestamp too :P